cvat/apps/engine/mixins.py

```diff
@@ -217,32 +236,55 @@ def append_tus_chunk(self, request, file_id):
                                       extra_headers={'Upload-Offset': tus_file.offset,
                                                      'Upload-Filename': tus_file.filename})
 
-    def validate_filename(self, filename):
+    def validate_uploaded_file_name(self, filename: str) -> bool:
```

Comment:
As long as you're renaming stuff, I think it would be better to call this something like `is_valid_uploaded_file_name`, to make it clear that it returns a boolean value.